### ISSUE TITLE
Do keywords collection in parallel

### ISSUE THREAD

**Issue by fridex** [fabric8-analytics/fabric8-analytics-tagger#1]
It takes ~12hours to gather keywords from PyPI, for other bigger ecosystems it can take even more. We could optimize our collectors to do collection in parallel in threads. As larger keywords collection is one-time thing, this is not high priority, but rather nice-to-have feature.
